### PR TITLE
New page in -PS to show hash key status

### PR DESCRIPTION
**Pull request by hokiepokedad**
This PR adds a new option/page to the print status screen that will show all configured keys and their current usage.  Simply type the H key and <ENTER> to switch to the Hash status page page.

### CODE REVIEW

File: pogom/search.py [hunk 1]
```diff
@@ -45,6 +45,8 @@
 
 from .proxy import get_new_proxy
 
+from pgoapi.hash_server import HashServer
```
Comment (chowderchow) [inline pogom/search.py]: should be in l39

Comment (hokiepokedad) [inline pogom/search.py]: fixed :)

File: pogom/search.py [hunk 2]
```diff
@@ -199,9 +204,28 @@ def status_printer(threadStatus, search_items_queue_array, db_updates_queue, wh_
...
+        elif display_type[0] == 'hashstatus':
+            status_text.append('-----------------------------------------')
+            status_text.append('Hash key status:')
+            status_text.append('-----------------------------------------')
+
+            status = '{:20} | {:9} | {:7}'
+            status_text.append(status.format('Key', 'Remaining', 'Maximum'))
+
+            for key in args.hash_key:
+				rm = 'Not active'
```
Comment (chowderchow) [inline pogom/search.py]: Please fix indentations for l217-225

Comment (hokiepokedad) [inline pogom/search.py]: python noob here :)  fixed

**Pull request merged**